index.js

```diff
@@ -34,9 +36,23 @@ const lint = options => {
 
 lint.report = async options => {
 	const spinner = ora('Linting').start();
+
+	try {
+		await lint._report(options, spinner);
+	} catch (error) {
+		spinner.fail(error.message);
+		process.exitCode = 1;
+	}
+};
+
+lint._report = async (options, spinner) => {
 	let temp = null;
 
 	if (isUrl(options.filename)) {
+		if (!rGithubRepo.test(options.filename)) {
```

Comment:
[`is-github-url`](https://github.com/alferov/is-github-url) should probably be used here instead, as it's properly tested and can handle different URL variations and structures.

Reply:
@itaisteinherz Great! I was wondering there had a package like that.